Password optional sign in javascript node guide (#3437)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/pwd-optional-new-sign-up-email/main/nodeexpress/integrationsteps.md

> ### 1: Add sign up link on sign-in page

Reviewer: ```suggestion
### 1: Add sign-up link on sign-in page
```

Add a **Sign up** link on your app's sign-in page. When the user clicks this link, they are redirected to a sign-up page where they can sign up for a new account.

<div class="common-image-format bordered-image">

> ![Screenshot showing a sign in page with a sign up link.](/img/pwd-optional/pwd-optional-sign-up-link-sign-in-page.png)

Reviewer: ```suggestion
![Screenshot showing a sign in page with a sign-up link.](/img/pwd-optional/pwd-optional-sign-up-link-sign-in-page.png)
```

Author: updated.

### 5. Display remaining optional authenticators

After the user verifies their identity using the email authenticator, `OktaAuth.idx.proceed()` returns an `IdxTransaction` object indicating that the user can enroll in additional optional authenticators including the password authenticator. The `IdxTransaction.nextStep.canSkip` property identifies whether or not the remaining authenticators are optional. In this case, the property is set to `true`.

```json
{
  status: "PENDING",
  nextStep: {
    name: "select-authenticator-enroll",
    options: [
      {
        label: "Okta Verify",
        value: "okta_verify",
      },
      {
        label: "Password",
        value: "okta_password",
      },
    ],
    canSkip: true,
  },
}
```

Create and display a page that lists the remaining optional authenticators and allows the user to skip registering any additional authenticators.

<div class="common-image-format bordered-image">

![Screenshot showing a page to skip the remaining optional authenticators.](/img/pwd-optional/pwd-optional-sign-up-authenticators-page.png)

</div>

> ### 6. Skip remaining optional authenticators and complete sign-up

Reviewer: ```suggestion
### 6. Skip remaining optional authenticators and complete sign-up flow
```

> When the user opts to skip registering any more authenticators, call `OktaAuth.idx.proceed()` passing in an object with a `skip` property equal to `true`.

Reviewer: ```suggestion
When the user opts to skip registering any more authenticators, call `OktaAuth.idx.proceed()` and pass in an object with a `skip` property equal to `true`.
```
